src/module/display-comments.js

```diff
@@ -0,0 +1,20 @@
+async function displayComments(id) {
+  const apiKey = 'tnE2k6P5BdZ2HCTjbd0V';
+  const response = await fetch(`https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${apiKey}/comments?item_id=${id}`,
+    {
+      method: 'GET',
+      headers: {
+        'Content-type': 'application/json; charset=UTF-8',
+      },
+    });
+
+  try {
+    const data = await response.json();
+    const commentsCount = document.getElementById(`comment${id}`);
+    commentsCount.textContent = `${data.length || ''}`;
+  } catch (err) {
```

Comment:
good job 👍, that you create a counter for comments and it's now mandatory or from the requirements of the project, but kindly check it again because it's not working as supposed. 😃

Reply:
I can't catch fetch response with status 400 in case if there are no comments for specific ID. So, I'm going to remove this part of code for a while.